templates/policies/openshift_cluster_capi_aws_cloud_credentials_policy.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
You should remove this file since it's no longer needed

Reply:
@vkareh  i did nt remove becuase i think we will be replacing machine api with this one in future so added it as placeholder

Reply:
Yes +1 to removing this